cloud/digital_ocean/digital_ocean.py

```diff
@@ -239,10 +245,11 @@ def setup(cls, api_token):
         cls.manager = DoManager(None, api_token, api_version=2)
 
     @classmethod
-    def add(cls, name, size_id, image_id, region_id, ssh_key_ids=None, virtio=True, private_networking=False, backups_enabled=False, user_data=None):
+    def add(cls, name, size_id, image_id, region_id, ssh_key_ids=None, virtio=True, private_networking=False, backups_enabled=False, user_data=None, ipv6=False):
         private_networking_lower = str(private_networking).lower()
         backups_enabled_lower = str(backups_enabled).lower()
-        json = cls.manager.new_droplet(name, size_id, image_id, region_id, ssh_key_ids, virtio, private_networking_lower, backups_enabled_lower,user_data)
+        ipv6_lower = str(ipv6).lower()
```

Comment:
Hello! Just a semi-random question from the author of the ["competing" PR](https://github.com/ansible/ansible-modules-core/pull/2364), implementing the same feature. :smile: 

I'm curious as to what is the point of lower-casing the value of `ipv6` argument, when it should receive only `True` or `False`? Besides, `dopy` [performs normalization of arguments](https://github.com/Wiredcraft/dopy/blob/master/dopy/manager.py#L78) before sending them off to DO API.

Best regards!